erpnext/stock/reorder_item.py

```diff
@@ -127,16 +127,21 @@ def _log_exception():
 				for d in items:
 					d = frappe._dict(d)
 					item = frappe.get_doc("Item", d.item_code)
+					uom = item.purchase_uom if request_type == 'Purchase' else item.stock_uom
+					conversion_factor = frappe.db.get_value("UOM Conversion Detail",
```

Comment:
Conversion factor should not be fetched at all, if request_type is not 'Purchase'. It should be 1.